mandatory/4-tax.js

```diff
@@ -4,8 +4,9 @@
   A business requires a program that calculates how much the price of a product is including sales tax
   Sales tax is 20% of the price of the product.
 */
-
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+	return price + (price / 100) * 20;
```

Comment:
This is very good, I like that you added the parenthesis around the calculation, you could add even more parenthesis to make it even clearer how the calculation is run 👍

Reply:
CYF have told us to use the Prettier extension in VS Code which automatically creates and removes parenthesis so I can't do anything about this.